### PR TITLE
RouteOptions: added `max_weight`

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

File: services-directions-models/src/test/resources/route_options_v5.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   "depart_at": "2021-02-02'T'02:02",
   "max_height": 1.5,
   "max_width": 1.4,
+  "max_weight": 2.9,
```
Comment (LukasPaczos) [inline services-directions-models/src/test/resources/route_options_v5.json:35]: Could you make this same change in https://github.com/mapbox/mapbox-java/blob/main/services-directions/src/test/resources/route_options_v5.json?

Comment (RingerJK) [reply services-directions-models/src/test/resources/route_options_v5.json:35]: nice catch. Added

File: services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java [hunk 2]
```diff
@@ -95,6 +95,7 @@ Call<DirectionsResponse> getCall(
     @Query("depart_at") String departAt,
     @Query("max_height") Double maxHeight,
     @Query("max_width") Double maxWidth,
+    @Query("max_weight") Double maxWeight,
```
Comment (LukasPaczos) [inline services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java:98]: We're missing a `MapboxDirections` test for this.

Comment (RingerJK) [reply services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java:98]: in progress...

Comment (RingerJK) [reply services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java:98]: added

**Review by LukasPaczos — APPROVED** [review]
LGTM

**Pull request merged**